quiz.js

```diff
@@ -1,168 +1,137 @@
+                buttons.push(tempBtn);
         })
 
+        let correctBtn = $('<button>')
+            .text(question.correct_answer)
+            .val(question.correct_answer)
+            .addClass('hollow button answer')
+            .click(() => validateAnswer(true));
+        buttons.push(correctBtn);
+        return buttons;
     }
-    //this fnction validates the users answer
-    function validateAnswer() {
-        //set userChoice to the value of the btn the user clicked
-        const userChoice = $(this).val();
+
+    function correctAnswerClicked() {
+        score++;
+        rightWrongEl.text(`RIGHT`);
+        if (cheer.pause) {
+            cheer.currentTime = 0;
+            cheer.play();
+        }
+    }
+
+    function inCorrectAnswerClicked() {
+        rightWrongEl.text(`WRONG: Answer is ${correctAnswer}`);
+        if (jeer.pause) {
+            jeer.currentTime = 0;
+            jeer.play();
+        }
+    } 
+
+    //this function validates the users answer
+    function validateAnswer(isCorrect) {
         $(".answer").attr("disabled", true)
-        //if user chooses the correct answer the score goes up 1 and "RIGHT" appears in the right/wrong div
-        if (userChoice === correctAnswer) {
-            score++;
-            rightWrongEl.text(`RIGHT`);
-            if (cheer.pause) {
-                cheer.currentTime = 0;
-                cheer.play();
-            }
-            //if user is wrong "WRONG" apers in the right/wrong div along with the correct answer
+
+        if (isCorrect) { 
+            correctAnswerClicked(); 
         } else {
-            rightWrongEl.text(`WRONG: Answer is ${correctAnswer}`);
-            if (jeer.pause) {
-                jeer.currentTime = 0;
-                jeer.play();
-            }
+            inCorrectAnswerClicked();
         }
-        //question index increases by 1
+
         questionIndex++;
-        //setTimeout function to allow 1 second before generating a new question
-        setTimeout(function () {
-            //calls sendApiRequest function
-            generateQuestion();
+        setTimeout(generateQuestion, 3000)
+    }
 
-        }, 3000)
+    function fix(str) {
+        var txt = document.createElement("textarea");
+        txt.innerHTML = str;
+        return txt.value;
+    }
 
+    function resetQuestionZone() {
+        answerZoneEl.empty();
+        rightWrongEl.empty();
+        cheer.pause();
+        jeer .pause();
+        cheer.currentTime = 0;
+        jeer .currentTime = 0;
     }
-    //this function is for when the game is over
+
     function endGame() {
-        //stores score in localStorage
         localStorage.setItem('mostRecentScore', score);
-        //send user to end-game.html page
         return window.location.assign("./end-game.html");
-
-
     }
 
-    //this on click event turns the jukebox on and off
     $("#jukebox").click(function () {
-        DOOM.play();
-        if (DOOM.play) {
```

Comment:
`DOOM.paused` is what you were looking for here. Would have worked just fine, just doing a bit more work than it needs to by resetting the click handler.